### DISCUSSION TITLE
Lifetime License Issues and Support Contact

### DISCUSSION THREAD

**Discussion by MusTon7997** [aclap-dev/video-downloadhelper#1279] [License & Purchase requests]
### What's your issue?

I realized that, despite purchasing a lifetime license more than 12 months ago, the extension now has certain limitations. How is this possible?

How can I contact support?

### The 10 first digits (only 10) of your license number or email address

antonioper

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
hi,
Please send us an email at licenses@downloadhelper.net.
